src/views/profile/index.tsx

```diff
@@ -439,32 +443,31 @@ function ProfileScreen() {
                                 color={ipctColors.borderGray}
                                 style={{
                                     right:
-                                        -Dimensions.get('screen').height * 0.25,
+                                        -Dimensions.get('screen').height * 0.33,
                                     alignSelf: 'center',
                                 }}
                             />
```

Comment:
I guess I missed this before,
our design has icons. Use them (as svgs), avoid using 3rd party icons.

Reply:
Fixed both the icons and the dynamic spacing.